src/Analyzer.Types/ExclusionsConfigurationDefinition.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Templates.Analyzer.Types
+{
+    /// <summary>
+    /// Gets or sets the ExclusionsConfigurationDefinition.
```

Comment:
Update summary (a class can't get or set something).  Something like `The exclusions defined in a <see cref="ConfigurationDefinition"/>`.